src/components/Datatable/BatchModule/BatchActions/BatchActions.tsx

```diff
@@ -80,7 +80,7 @@ const BatchActions: React.FC<BatchActionsProps> = ({ actions }) => {
                 <Inline align="center" gap={SpaceSizes.xs}>
                   <span>{action.label}</span>
                   <Icon
-                    color={ColorTypes.neutral700}
+                    color={ColorTypes.primary600}
```

Comment:
We should drop the `color` prop entirely. Icon component inherits parent color so it should be inheriting from the button

Reply:
removed